src/rai_extensions/rai_open_set_vision/launch/example_communication_launch.xml

```diff
@@ -0,0 +1,33 @@
+<!--
+  ~ Copyright 2024 Robotec.ai
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<launch>
+    <arg name="dino_weights_path" default = ""/>
+    <arg name="sam_weights_path" default = ""/>
+    <arg name="image_path" default="$./src/rai_extensions/rai_open_set_vision/images/sample.jpg" />
```

Comment:
This script does not work with the default arguments. I think this is caused by the fact the script is ran from different location (rai/install) and not rai/. I think that there are two ways to handle this:  fix this the image should be either copied to install catalog or the default value should be dropped.